src/app/components/Item/ItemsContainer.jsx

```diff
@@ -146,10 +146,6 @@ class ItemsContainer extends React.Component {
     let itemsToDisplay = [...this.state.items];
     let pagination = null;
 
-    if (!itemsToDisplay || !itemsToDisplay.length) {
```

Comment:
This is no longer needed because a bib with items can have no items after running a search or filter request.